.github/workflows/font.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Featherity
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install 'FontCustom'
+        run: |
+          echo "Dependencies will be installed, no logs will be displayed... Wait a moment"
+          sudo apt-get install zlib1g-dev fontforge woff2 > /dev/null 2>&1
+          git clone https://github.com/bramstein/sfnt2woff-zopfli.git sfnt2woff-zopfli > /dev/null 2>&1
+          cd sfnt2woff-zopfli
+          make > /dev/null 2>&1
+          sudo mv sfnt2woff-zopfli /usr/local/bin/sfnt2woff
+          git clone --recursive https://github.com/google/woff2.git > /dev/null 2>&1
+          cd woff2
+          sudo make clean all > /dev/null 2>&1
+          sudo mv woff2_compress /usr/local/bin/ && sudo mv woff2_decompress /usr/local/bin/
+          sudo gem install fontcustom > /dev/null 2>&1
+          cd /home/runner/work/featherity/featherity
```

Comment:
Is this not better to split those commands up in separate commands?
In case if one of the commands fail, we can directly see where goes wrong.

Reply:
> Hi @yanquisalexander,
> 
> Awesome work, looks really good!
> What's the exact output of the zip file?

- Featherity.css
- Featherity.eot
- Featherity.svg
- Featherity.ttf
- Featherity.woff
- Featherity.woff2
- Featherity-preview.html

---

> Is this not better to split those commands up in separate steps?
> In case if one of the commands fail, we can directly see where goes wrong.


I will edit to show the logs, anyway if something fails the execution stops automatically and shows the error logs